kafka_consumer/datadog_checks/kafka_consumer/client/kafka_client.py

```diff
@@ -5,17 +5,8 @@
 
 
 class KafkaClient(ABC):
-    def __init__(self, check, tls_context) -> None:
-        self.check = check
-        self.log = check.log
-        self._kafka_client = None
-        self._highwater_offsets = {}
-        self._consumer_offsets = {}
-        self._context_limit = check._context_limit
-        self._tls_context = tls_context
-
-    def should_get_highwater_offsets(self):
```

Comment:
I wanted to move this back into the check logic since it's not too relevant for a kafka client to do.